[DOCS] Corrected and simplified CTAs for getting customer support (#9492)
great-expectations/great_expectations: docs/docusaurus/versioned_docs/version-0.18/resources/get_support.md

## Try solving your issue first

> The following are some recommendations for resolving issues on your own. Resolving issues on your own builds your own knowledge and provides you with an opportunity to share that knowledge with the community.

Reviewer: ```suggestion
The following are some recommendations for resolving issues on your own. Resolving issues on your own builds your knowledge and allows you to share that knowledge with the community.
```

### Search the GX documentation

Search the docs you're using currently for an answer to your issue or question.

If you're new to GX Cloud, review [About GX Cloud](/cloud/about_gx.md) or [Try GX Cloud](/cloud/try_gx_cloud.md).

> If you're new to GX OSS, try the [Quickstart](/oss/tutorials/quickstart.md) or one of the specific getting started guides such as [Get started with Great Expectations and Databricks](/oss/tutorials/getting_started/how_to_use_great_expectations_in_databricks.md), or [Get started with Great Expectations and SQL](/oss/tutorials/getting_started/how_to_use_great_expectations_with_sql.md).

Reviewer: ```suggestion
If you're new to GX OSS, try the [Quickstart](/oss/tutorials/quickstart.md) or one of the specific getting started guides such as [Get started with Great Expectations and Databricks](/oss/tutorials/getting_started/how_to_use_great_expectations_in_databricks.md) or [Get started with Great Expectations and SQL](/oss/tutorials/getting_started/how_to_use_great_expectations_with_sql.md).
```

## Ask the right questions and provide the right information

> Taking the time to formulate your questions properly and providing all the relevant background information can save time and improve response times. The following are some recommendations that will help the GX Developer Advocates (DAs) and other community members help you.

Reviewer: ```suggestion
Formulating your questions properly and providing all the relevant background information can save time and improve response times. The following recommendations will help the GX Developer Advocates (DAs) and other community members assist you.
```

- A detailed explanation of what you're trying to resolve including what attempts you've made on your own to date.

> ## GX Cloud Support

Reviewer: ```suggestion
## GX Cloud support
```

> - If you've encountered an error you can't resolve, first try creating a new topic on the [GX Discourse - GX Cloud Support](https://discourse.greatexpectations.io/c/oss-support/11) forum. This is the recommended option as answers aren't removed after 90 days, and you're likely to get a quicker response.

Reviewer: ```suggestion
- Create a new topic on the [GX Cloud Discourse forum](https://discourse.greatexpectations.io/c/cloud-support/17) if you've encountered an error you can't resolve. This is the recommended option, as answers aren't removed after 90 days, and you're likely to get a quicker response.
```

> - You can also email us at support@greatexpectations.io or create a ticket on our [support portal](https://support.greatexpectations.io).

Reviewer: ```suggestion
- Send an email to support@greatexpectations.io or create a ticket on the [support portal](https://support.greatexpectations.io).
- Discuss GX Cloud issues on the [GX Cloud Slack community channel](https://greatexpectationstalk.slack.com/archives/C051D941XAL).
```

> ## GX OSS Support

Reviewer: ```suggestion
## GX OSS support
```

> - If you want to chat with community members about general issues and solutions, join the [GX Slack community](https://greatexpectationstalk.slack.com/archives/CUTCNHN82). To discuss GX Cloud issues, join the [GX Cloud Slack community](https://greatexpectationstalk.slack.com/archives/C051D941XAL).

Reviewer: ```suggestion
- Chat with community members about general issues and seek solutions on the [GX Slack community channel](https://greatexpectationstalk.slack.com/archives/CUTCNHN82). 
```
